devel/ansible/roles/bodhi/files/.bashrc

```diff
@@ -36,6 +36,16 @@ function btest {
     bteststyle && bdocs && py.test-3 $@ /home/vagrant/bodhi/bodhi/tests && diff-cover /home/vagrant/bodhi/coverage.xml --compare-branch=develop --fail-under=100
 }
 
+alias bdestroyservices="sudo docker-compose -f /home/vagrant/bodhi/devel/docker/compose-services.yml down --rmi all -v"
+
+function brunservices {
+    pushd /tmp;
+    curl -o waiverdb.dump.xz https://infrastructure.fedoraproject.org/infra/db-dumps/waiverdb.dump.xz
+    xz -d --keep --force waiverdb.dump.xz
+    popd;
+    sudo docker-compose -f /home/vagrant/bodhi/devel/docker/compose-services.yml up -d
+}
```

Comment:
We could just make this all be part of bstop and bstart - what do you think? I don't mind it being separate, but maybe we should pick a less generic name if we do keep it separate. Maybe bstopdeps and bstartdeps or similar?

Reply:
> We could just make this all be part of bstop and bstart - what do you think?

I'm not sure about that. When I test my changes made in Bodhi, I restart(by bstop/bstart/brestart)
Bodhi many times, but external services(GW + WDB) does not need to be stopped/restarted/stopped each time.

> I don't mind it being separate, but maybe we should pick a less generic name if we do keep it separate. Maybe bstopdeps and bstartdeps or similar?

The name of the command should describe what command do, so first command creates services +  downloads db dumps + run dockers, and the second removes db dumps + removed docker services + removes images(to fetch latest each time). I'm ok with "bstopdeps" and "bstartdeps", but let's add another bremovedeps.
So:
- bstartdeps -> will create services +  download db dumps + run dockers
- brestartdeps -> (do we really need it? configuration changes and so on)
- bstopdeps -> will stop docker service
- bremovedeps -> will stop docker service + remove db dumps + remove docker services + removes images

What do you think?

Reply:
Yeah I think that argument about frequently restarting Bodhi and not wanting that to spend the time of restarting deps does make sense.

I don't think we need a bremovedeps. I also think it does make sense for the vagrant setup to run the deps by default without the user having to type bstartdeps - we already do that for bodhi itself, bodhi's consumer, rabbitmq, the database, etc.

Reply:
All that said, a bremovedeps is fine since you already have it. Doesn't hurt anything.

Reply:
Looks to me like this is resolved.